generated/nifpga/nifpga_service.cpp

```diff
@@ -483,13 +483,13 @@ ::grpc::Status MonikerReadArrayBool(void* data, google::protobuf::Arena& arena,
 ::grpc::Status MonikerReadArrayDbl(void* data, google::protobuf::Arena& arena, google::protobuf::Any& packedData)
 {
     MonikerReadArrayDblData* function_data = static_cast<MonikerReadArrayDblData*>(data);
-    auto library = function_data->library;    
+    auto library = function_data->library;
```

Comment:
you should look at this file and the mako and make sure changes done right now in the mako should not affect FPGA service file. These are just changes related to space and extra line; you should update the mako accordingly.

Reply:
Removed extra added spaces. The changes present in nifpga_service.cpp right now are correct as unnecessary trailing whitespaces have been removed as part of that.